Tracker/Controllers/CategoriesViewController.swift

```diff
@@ -79,31 +77,42 @@ final class CategoriesViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        viewModel.$selectedCategoryName.bind { [weak self] _ in
+            guard let self = self else { return }
+            self.delegate?.didSelectCategory(with: self.viewModel.selectedCategoryName)
+            self.tableView.reloadData()
+        }
+        
         viewModel.$categories.bind { [weak self] _ in
             guard let self = self else { return }
+            if self.viewModel.categories?.count != 0 {
+                self.heightTableView = self.viewModel.categories!.count * 75 - 1
+                self.tableViewHeightConstraint!.constant = CGFloat(self.heightTableView)
```

Comment:
Один знак меняем, а уже к крашу не приведем
```suggestion
                self.tableViewHeightConstraint?.constant = CGFloat(self.heightTableView)
```